### ISSUE TITLE
Populate and polish website

### ISSUE THREAD

**Issue by emdupre** [nilearn/dev-days-2021#10]
The current website has initial populated content, but it could use a few updates : 

- [ ] The individual talks for last year's speakers should all be linked out from the [Nilearn Events Youtube channel](https://www.youtube.com/channel/UCU6BMAi2zOhNFnDkbdevmPw)
- [ ] The sprint infrastructure content should be added as it is decided
- [ ] The wording should be adjusted throughout, particularly on the main page re: the motivation for this year's dev days !

**Comment by NicolasGensollen** [comment]
Not sure this is what you had in mind, but I added the links to the youtube videos for the last year talks (see 4ba3839)

**Comment by bthirion** [comment]
Thx!

**Comment by emdupre** [comment]
Thanks, @NicolasGensollen ! I went ahead and removed the scheduling information for last year's talks in 486868744112a7399281d8888afb23a0eb7c9102. I had added this year's in fcdc8c7a77c5874967d4b504384d4af7558172dd, and I imagine we don't need both :)

**Comment by emdupre** [comment]
@bthirion should we adjust the wording through to highlight Nibabel as well ?

**Comment by bthirion** [comment]
Indeed. @NicolasGensollen can you do at least a pass on that before we start to communicate ?
Thx,

**Comment by NicolasGensollen** [comment]
I changed the wording, and added a few pointers to Nibabel resources in 406eb73. 
I'm not sure at what point we want to integrate Nibabel here though... I changed the page title, but not the logo for example. Feel free to undo some changes if needed.
